<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>usergroupManage</class>
 <widget class="QWidget" name="usergroupManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>25</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>用户信息</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>341</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: none;
background-color: rgb(255, 255, 255);
gridline-color: rgb(130, 135, 144);
border-color: rgb(130, 135, 144);
border-width: 1px;border-style: solid;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addpushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>137</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
background-image: url(:/monres/useradd.bmp);
border-color: rgb(255, 255, 255);
border-width: 1px;
border-style: solid;}
QPushButton:pressed
{background-color: rgb(86, 164, 246);color: rgb(255, 255, 255);border-color: rgb(170, 170, 170);border-width: 2px;border-style: solid;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="deletepushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>137</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
background-image: url(:/monres/userdel.bmp);
border-color: rgb(255, 255, 255);
border-width: 1px;
border-style: solid;}
QPushButton:pressed
{background-color: rgb(86, 164, 246);color: rgb(255, 255, 255);border-color: rgb(170, 170, 170);border-width: 2px;border-style: solid;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>80</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>超级管理员</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>120</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>管理员用户</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>160</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>操作员用户</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>200</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>245</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>密    码:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>290</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>确    认:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernamelineEdit">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>195</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit:focus{background-color: rgb(204, 232, 207);}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwdlineEdit">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>240</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit:focus{background-color: rgb(204, 232, 207);}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="surelineEdit">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>285</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit:focus{background-color: rgb(204, 232, 207);}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="savepushButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>340</y>
     <width>137</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
background-image: url(:/monres/usersave.bmp);
border-color: rgb(255, 255, 255);
border-width: 1px;
border-style: solid;}
QPushButton:pressed
{background-color: rgb(86, 164, 246);color: rgb(255, 255, 255);border-color: rgb(170, 170, 170);border-width: 2px;border-style: solid;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>35</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>用户列表</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>901</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>lineEdit</zorder>
  <zorder>label_2</zorder>
  <zorder>addpushButton</zorder>
  <zorder>deletepushButton</zorder>
  <zorder>radioButton</zorder>
  <zorder>radioButton_2</zorder>
  <zorder>radioButton_3</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>usernamelineEdit</zorder>
  <zorder>passwdlineEdit</zorder>
  <zorder>surelineEdit</zorder>
  <zorder>savepushButton</zorder>
  <zorder>label_7</zorder>
  <zorder>tableWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
